changelog/_unreleased/2025-04-12-entityindexerregistry-extension-indexer-only.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Add possibility to the context to define which specific indexer should run
+author: Niel Duysters
+author_email: niel.duysters@meteor.be
+author_github: @nielduysters
+---
+# Core
+* Added `EntityIndexerRegistry::EXTENSION_INDEXER_ONLY` to the `EntityIndexerRegistry` to allow specific indexers.
+
+# Upgrade Information
+
+## Add possibility to specify indexer in tontext
+< Add some information and how to use it here>
```

Comment:
of course you need to fill that with life :grimacing: that was just a placeholder from my side